pyiron/lammps/control.py

```diff
@@ -101,23 +101,60 @@ def load_default(self, file_content=None):
         self.load_string(file_content)
 
     def calc_minimize(
-        self, e_tol=0.0, f_tol=1e-8, max_iter=100000, pressure=None, n_print=100
+        self,
+        e_tol=0.0,
+        f_tol=1e-4,
+        max_iter=100000,
+        pressure=None,
+        n_print=100,
+        style='cg'
     ):
-        max_evaluations = 10 * max_iter
+        """
+        Sets parameters required for minimization.
+
+        Args:
+            e_tol (float): If the magnitude of difference between energies of two consecutive steps is lower than or
+                equal to `e_tol`, the minimisation terminates. (Default is 0.0 eV.)
+            f_tol (float): If the magnitude of the global force vector at a step is lower than or equal to `f_tol`, the
+                minimisation terminates. (Default is 1e-4 eV/angstrom.)
+            max_iter (int): Maximum number of minimisation steps to carry out. If the minimisation converges before
+                `max_iter` steps, terminate at the converged step. If the minimisation does not converge up to
+                `max_iter` steps, terminate at the `max_iter` step. (Default is 100000.)
+            pressure (float/list/tuple/numpy.ndarray): Pressure in GPa at which minimisation is to be carried out. If
+                None, isochoric (constant volume) condition will be used. If a float, cell shape changes are allowed.
+                in each of the three primary axes, but no shearing occurs. If array-like, (up to) the first six
+                elements will be interpreted as the x, y, z, xy, xz, and yz components of the pressure tensor,
+                respectively. If this component-wise mode is used, cell changes corresponding to any one element of the
+                stress tensor can be selectively disabled by setting this element to None. (Default is None, run
+                isochorically.)
+            n_print (int): Write (dump or print) to the output file every n steps (Default: 100)
+            style ('cg'/'sd'/other values from Lammps docs): The style of the numeric minimization, either conjugate
+                gradient, steepest descent, or other keys permissible from the Lammps docs on 'min_style'. (Default
+                is 'cg' -- conjugate gradient.)
+        """
+        # This docstring is a source for the calc_minimize method in pyiron.lammps.base.LammpsBase.calc_minimize and
+        # pyiron.lammps.interactive.LammpsInteractive.calc_minimize -- Please ensure that changes to signature or
+        # defaults stay consistent!
+        max_evaluations = 100 * max_iter
+        GPA_TO_BAR = spc.giga * 1.0 / spc.bar
+        if self["units"] != "metal":
```

Comment:
Is there a reason why you restrict `calc_minimize()` only for calculations with `metal` units? Currently, pyiron users also use LAMMPS with `real` units (water calculations for example). So maybe a better way to do this is to define something like

```
pressure_conv = {"metal":  spc.giga * 1.0 / spc.bar, "real":  spc.giga * 1.0 / spc.atm} 
if self["units"] not in ["metal", "real"]:
    raise NotImplementedError("`calc_md()` does not support the {} LAMMPS unit yet!".format(self["units"]))
``` 

And then later on

"""
str_press += str_axis + str(press * pressure_conv[self["units"]])
"""

Reply:
The reason was because I was lazy and didn't want to look too closely at the units :p ...Now I did [look more closely](https://github.com/pyiron/pyiron/issues/465) ;)

This restriction is lifted here though, and the conversion dictionaries are typed out for using in a bit more thorough treatment of units elsewhere in the future.